treeherder/model/fixtures/repository.json

```diff
@@ -1589,5 +1589,33 @@
       "description": "",
       "tc_root_url": "https://firefox-ci-tc.services.mozilla.com"
     }
+  },
+  {
+    "pk": 114,
+    "model": "model.repository",
+    "fields": {
+      "dvcs_type": "hg",
+      "name": "mozilla-esr78",
+      "url": "https://hg.mozilla.org/releases/mozilla-esr78",
+      "active_status": "active",
+      "codebase": "gecko",
+      "repository_group": 2,
+      "description": "",
+      "tc_root_url": "https://firefox-ci-tc.services.mozilla.com"
+    }
+  },
+  {
+    "pk": 115,
+    "model": "model.repository",
+    "fields": {
+      "dvcs_type": "hg",
+      "name": "comm-esr78",
+      "url": "https://hg.mozilla.org/releases/comm-esr78",
+      "active_status": "onhold",
```

Comment:
Is this to be active or onhold?

Reply:
Sounded like this was a placeholder for later based on PR description.